gallery/src/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import packageJson from '../../package.json';
+
+const plaidVillain = require('./plaid-villain.svg');
+const tartanLogotype = require('./tartan-logotype.svg');
+
+import './PageHeader.scss';
+
+const PageHeader = () => (
+  <header className="nx-gallery-page-header">
+    <div className="nx-gallery-page-header__inner">
+      <h1 className="nx-gallery-page-header__logo">
+        <a className="nx-gallery-page-header__home-link"
```

Comment:
Exactly how much of the header is supposed to be the home link?

Reply:
> Exactly how much of the header is supposed to be the home link?

Talked it over with Jenni, and updated it here: c400966dcd9ada6ecae2cc9d764dcb3363e9fe22